common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -629,78 +629,120 @@ def test_navigation_links(self):
-        self.browse_teams_page = BrowseTeamsPage(self.browser, self.course_id, self.topic)
-        self.browse_teams_page.visit()
-        self.create_team_page = CreateTeamPage(self.browser, self.course_id, self.topic)
-        self.team_name = 'Avengers'
+    topic = {'name': 'Example Topic', 'id': 'example_topic', 'description': 'Description'}
+    team_name = 'Avengers'
 
-    def verify_page_header(self):
+    def verify_page_header(self, title, description, breadcrumbs):
         """
         Verify that the page header correctly reflects the
         create team header, description and breadcrumb.
         """
-        self.assertEqual(self.create_team_page.header_page_name, 'Create a New Team')
-        self.assertEqual(
-            self.create_team_page.header_page_description,
-            'Create a new team if you can\'t find existing teams to join, '
-            'or if you would like to learn with friends you know.'
-        )
-        self.assertEqual(self.create_team_page.header_page_breadcrumbs, self.topic['name'])
+        self.assertEqual(self.create_or_edit_team_page.header_page_name, title)
+        self.assertEqual(self.create_or_edit_team_page.header_page_description, description)
+        self.assertEqual(self.create_or_edit_team_page.header_page_breadcrumbs, breadcrumbs)
 
     def verify_and_navigate_to_create_team_page(self):
         """Navigates to the create team page and verifies."""
         self.browse_teams_page.click_create_team_link()
-        self.verify_page_header()
+        self.verify_page_header(
+            title='Create a New Team',
+            description='Create a new team if you can\'t find existing teams to '
+                        'join, or if you would like to learn with friends you know.',
+            breadcrumbs=self.topic['name']
+        )
+
+    def verify_and_navigate_to_edit_team_page(self):
+        """Navigates to the edit team page and verifies."""
+        # pylint: disable=no-member
+        self.assertEqual(self.team_page.team_name, self.team['name'])
+        self.assertTrue(self.team_page.edit_team_button_present)
+
+        self.team_page.click_edit_team_button()
+
+        self.create_or_edit_team_page.wait_for_page()
 
-    def fill_create_form(self):
-        """Fill the create team form fields with appropriate values."""
-        self.create_team_page.value_for_text_field(field_id='name', value=self.team_name, press_enter=False)
-        self.create_team_page.value_for_textarea_field(
+        # Edit page header.
+        self.verify_page_header(
+            title='Edit Team',
+            description='If you edit any team details, you should notify team members of your changes.',
+            breadcrumbs=self.topic['name']
+        )
+
+    def verify_form_values_on_team_page(self, name, description, location, language):
+        """Verify the values for create/edit form."""
```

Comment:
I think this doc string is wrong-- the create/edit form isn't what is being tested.

Can this method just be verify_team or verify_team_values?